src/patterns/by-the-numbers.scss

```
@@ -1,228 +1,275 @@
 .wmf-pattern-by-the-numbers {
+	@media only screen and ( min-width: 782px ) {
+		padding-bottom: 0 !important;
+	}
 
-	.wmf-pattern-by-the-numbers__masonry {
-		margin-top: 1rem;
+	.wmf-pattern-most-visited,
+	.wmf-pattern-ads-served,
+	.wmf-pattern-most-editors,
+	.wmf-pattern-two-columns,
+	.wmf-pattern-three-columns,
+	.wmf-pattern-piechart,
+	.wmf-pattern-views {
+		margin: 0 auto 4rem auto;
+
+		&__text {
+			line-height: 1.3;
+		}
+	}
 
-		> .wp-block-group__inner-container {
-			margin-bottom: 2rem;
-			margin-left: 10%;
-			margin-right: 10%;
+	.wmf-pattern-most-visited,
+	.wmf-pattern-ads-served {
 
-			> .wp-block-group {
-				margin-top: 4rem;
-			}
+		margin-left: auto;
+		margin-top: 7rem;
+		margin-right: auto;
+		position: relative;
+		text-align: center;
+		max-width: 330px;
+
+		.wp-block-group__inner-container {
+			height: auto !important;
+			position: relative;
+			z-index: 2;
 		}
 
-		.masonry-layout {
-			box-sizing: border-box;
-			display: grid;
-			grid-template-columns: repeat(2, 1fr);
-			gap: 8rem;
-			margin-top: 2rem;
+		&__number {
+			font-size: 179px;
+			font-weight: 700;
+			line-height: 1;
+			padding-top: 6rem;
+			margin-bottom: 0;
 		}
 
-		.masonry-layout > div > img,
-		.masonry-layout > div > div {
-			width: 100%;
-			margin-bottom: 6rem;
+		&__text {
+			font-size: $font-size-xlarge;
+			margin: 0 auto 0 auto;
 		}
 
-		.masonry-layout > div > div:last-of-type {
-			margin-bottom: 2rem;
+		figure,
+		svg {
+			height: auto !important;
+			left: 0;
+			position: absolute;
+			top: 0;
+			width: 100%;
+			z-index: 1;
 		}
 
-		.masonry-column-1 {
-			padding-left: 4rem;
+		img {
+			margin: auto;
 		}
+	}
 
-		.masonry-column-2 {
-			padding-right: 4rem;
+	.wmf-pattern-most-visited {
+		margin-top: 0;
+		margin-bottom: 3rem;
+
+		svg {
+			width: 170% !important;
 		}
+	}
 
-		// stylelint-disable-next-line no-descending-specificity
-		img {
-			margin: auto;
+	.wmf-pattern-most-editors {
+		text-align: center;
+		max-width: 330px;
+
+		&__number {
+			font-size: 88px;
```

I know we're bending some rules but let's keep trying to use rem for font sizes where we can (suggestion applies to a number of lines)
```suggestion
			font-size: 5.5rem; // 88px
```